schema/data/prod/json_schema/emission_intensity_report.json

```diff
@@ -62,7 +61,7 @@
         "title": "GHG Emission Intensity Performance",
         "type": "number"
       },
-      "calculatedPaymentPercentage": {
+      "paymentPercentage": {
```

Comment:
As far as I can tell, `paymentPercentage` and `actualPerformanceMilestoneAmount` aren't saved in any tables